### PR TITLE
Facebook

### PR DESCRIPTION
**Pull request by ledamint**

### CODE REVIEW

File: src/render/telegram.tsx [hunk 1]
```diff
@@ -6,17 +6,30 @@ import { App } from '../App';
...
+const isDevelopment = process.env.NODE_ENV === 'development';
+
 if (process.env.TELEGRAM_TOKEN === undefined) {
     throw new Error('Provide TELEGRAM_TOKEN to .env https://core.telegram.org/bots#6-botfather');
 }
 
-const urbanBotTelegram = new UrbanBotTelegram(process.env.TELEGRAM_TOKEN, {
-    polling: true,
-});
+let options;
```
Comment (oTkPoBeHuE) [inline src/render/telegram.tsx:15]:
```ts

function getOptions() {
    if (isDevelopment) {
        return {
            polling: true,
        };
    }

    return {
        webHook: {
            port: process.env.PORT ? Number(process.env.PORT) : undefined,
        },
    }
}

const urbanBotTelegram = new UrbanBotTelegram(process.env.TELEGRAM_TOKEN,  getOptions());

File: src/render/facebook.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+if (process.env.FACEBOOK_PAGE_ACCESS_TOKEN === undefined) {
+    throw new Error('Provide FACEBOOK_PAGE_ACCESS_TOKEN to .env https://developers.facebook.com/apps/');
+}
+
+const urbanBotFacebook = new UrbanBotFacebook({
+    appSecret: process.env.FACEBOOK_APP_SECRET,
+    pageAccessToken: process.env.FACEBOOK_PAGE_ACCESS_TOKEN,
+    verifyToken: process.env.FACEBOOK_VERIFY_TOKEN,
+    port: process.env.PORT ? Number(process.env.PORT) : undefined,
```
Comment (oTkPoBeHuE) [inline src/render/facebook.tsx:21]: Maybe we should provide String in Urban's bots?.
``` ts
function parseNumber(numberValue: number | string) {
  if(typeof numberValue === "number"){
    return numberValue;
  }
  const parsedValue = parseInt(numberValue);
  if(isNaN(parsedValue)){
    throw new Error("Incorrect number")
 }
  return parsedValue;
}

try {
 return port ? parseNumber(port) : undefined;
} catch(e) {
  throw new Error("Error converting port string to number.")
}
```

Comment (ledamint) [reply src/render/facebook.tsx:21]: Port is always number, so I guess number type is better

File: src/render/telegram.tsx (hunk 1 above)
Comment (ledamint) [reply src/render/telegram.tsx]: fixed

**Pull request merged**